autoload/project_root.vim

```diff
@@ -28,37 +16,50 @@ endfunction
 function! project_root#find()
     if !get(g:, 'JavaComplete_MavenRepositoryDisabled', 0)
         if !exists('g:JavaComplete_PomPath')
-            let g:JavaComplete_PomPath = project_root#find_file('pom.xml')
-            if g:JavaComplete_PomPath != ""
-                return fnamemodify(g:JavaComplete_PomPath, ':p')
+            if filereadable(getcwd() . g:utils#FILE_SEP . "pom.xml")
+                let g:JavaComplete_PomPath = getcwd() . g:utils#FILE_SEP . "pom.xml"
+            else
+                let g:JavaComplete_PomPath = project_root#find_file('pom.xml')
```

Comment:
Actually, we only need to add flag argument to `find_file` function. For example: `project_root#find_file('pom.xml', 1)`